driver-core/src/main/java/com/datastax/driver/core/SchemaParser.java

```diff
@@ -22,6 +22,7 @@
 import static com.datastax.driver.core.SchemaElement.TYPE;
```

Comment:
Please add "Modified by" at the top, at line 16 (not needed in other file, as it already has this notice from before):

```
/*
 * Copyright (C) 2021 ScyllaDB
 *
 * Modified by ScyllaDB
 */
```

Reply:
done